### PR TITLE
Create .gitpod.yml

### PR DESCRIPTION
**Pull request by faddat**
Closes: #807 

## What is the purpose of the change

* provide a consistent developer environment that does not require dependency management, even for rocksdb, cleveldb, and rust
* allow developers with m1 macs to work on osmosis confidently

## Brief change log

* adds a .gitpod.yml file to osmosis

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (yes / no)
    -  no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (yes / no)
    - yes 
  - How is the feature or change documented? (not applicable   /   specification (`x/<module>/spec/`)  /  [Osmosis docs repo](https://github.com/osmosis-labs/docs)   /   not documented)

### CODE REVIEW

**Comment by p0mvn** [comment]
Hi @faddat . What is the status of this?

**Comment by faddat** [comment]
Done :D

**Comment by faddat** [comment]
It puts devs in a one-click paradise:

https://github.com/faddat/archlinux-docker/blob/master/cosmos/Dockerfile

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+image: ghcr.io/faddat/cosmos
+
+
+
+
```
Comment (p0mvn) [inline .gitpod.yml:5]:
```suggestion
```

**Review by p0mvn — APPROVED** [review]

**Pull request merged**